src/app/games/trexo/tests/trexo-half-piece.component.spec.ts

```diff
@@ -0,0 +1,216 @@
+/* eslint-disable max-lines-per-function */
+import { DebugElement } from '@angular/core';
+import { fakeAsync } from '@angular/core/testing';
+import { Coord } from 'src/app/jscaip/Coord';
+import { Coord3D } from 'src/app/jscaip/Coord3D';
+import { SimpleComponentTestUtils } from 'src/app/utils/tests/TestUtils.spec';
+import { TrexoHalfPieceComponent } from '../trexo-half-piece.component';
+import { TrexoComponent } from '../trexo.component';
+import { TrexoMove } from '../TrexoMove';
+
+describe('TrexoHalfSquareComponent', () => {
+
+    let testUtils: SimpleComponentTestUtils<TrexoHalfPieceComponent>;
+
+    let component: TrexoHalfPieceComponent;
+
+    /**
+     *     0------1
+     *    /      /|
+     *   2------3 |
+     *   |      | 4
+     *   |      |/
+     *   5------6
+     */
+
+    beforeEach(fakeAsync(async() => {
+        testUtils = await SimpleComponentTestUtils.create(TrexoHalfPieceComponent);
+        component = testUtils.getComponent();
+        component.coord = new Coord3D(1, 1, 0);
+        component.mode = TrexoComponent.modeMap['3D'];
+    }));
+
+    it('should create', fakeAsync(async() => {
+        // wait for the chat to be initialized (without it, ngOnInit will not be called)
```

Comment:
quoi ? Nos tests de sous-composant chargent le chat ? oO

Reply:
OEF, j'me souvenait même pas de ce copier coller là de "should create" :D